### PR TITLE
Add settings for opening Duck Player in new tab

### PR DESCRIPTION
**Pull request by CrisBarreiro**
Task/Issue URL: https://app.asana.com/0/488551667048375/1208500747095741/f

### Steps to test this PR

_Feature 1_
- [x] Open Duck Player settings
- [x] Check open in new tab
- [x] Close Duck Player settings
- [x] Open then again
- [x] Check the setting is still enabled

### UI changes
Screenshot_20241008_145837

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208500747095741

### CODE REVIEW

**Comment by CrisBarreiro** [comment]
* **#5110** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5110?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5109** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5109?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CrisBarreiro and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: duckplayer/duckplayer-impl/src/main/res/layout/activity_duck_player_settings.xml [hunk 1]
```diff
@@ -125,6 +125,13 @@
                 app:primaryText="Open YouTube Videos in Duck Player"
                 />
 
+            <com.duckduckgo.common.ui.view.listitem.OneLineListItem
+                android:id="@+id/openDuckPlayerInNewTabToggle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:primaryText="Open Duck Player in a new tab whenever possible"
```
Comment (joshliebe) [inline duckplayer/duckplayer-impl/src/main/res/layout/activity_duck_player_settings.xml:132]: Will this be translated?

**Review by joshliebe — APPROVED** [review]
Only one [comment](https://github.com/duckduckgo/Android/pull/5109#discussion_r1793765300). Otherwise tested, LGTM ✅

File: duckplayer/duckplayer-impl/src/main/res/layout/activity_duck_player_settings.xml (hunk 1 above)
Comment (CrisBarreiro) [reply duckplayer/duckplayer-impl/src/main/res/layout/activity_duck_player_settings.xml:132]: Just extracted it to donottranslate and will now open a translations PR

**Pull request merged**